CHANGES.md

```diff
@@ -15,7 +15,23 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- DGGS: Add the new `components/dggs` providing an OGC API for Discrete Global Grid Systems.
+  - DGGS API available through `/dggs-api` path (default, configurable via `DGGS_API_PATH`).
+  - Redirects available for `/ogcapi/dggs/...` and  `/ogcapi/collections/.../dggs/...`.
+  - Sample configuration (minimum 1 resolvable data provider required) uses the new
+    feature of `optional-components/secure-data-proxy` on CRIM's Hirondelle server.
+
+- Data: Allow `optional-components/secure-data-proxy` to define generic and flexible locations.
+  - `SECURE_DATA_PROXY_ROOT` can be defined as mount directory inside the `proxy` service.
+  - `SECURE_DATA_PROXY_LOCATIONS` can be defined with any amount of custom locations.
+  - All locations can be configured (as desired) under Magpie `secure-data-proxy` service for access control.
+  - Other components (`wps_output-volume`, `stac-data-proxy`) that can optionally use this security middleware
+    via `SECURE_DATA_PROXY_AUTH_INCLUDE` can still do so. Their mount points are handled separately.
```

Comment:
Does using `SECURE_DATA_PROXY_AUTH_INCLUDE` var means component `secure-data-proxy` must be enabled?  Should mention it clearly, otherwise it does work and users won't know why.

Reply:
It's the other way around. By enabling `secure-data-proxy`, `SECURE_DATA_PROXY_AUTH_INCLUDE` gets defined with the nginx definition containing the pre-request to perform auth checks. All nginx configs that used the variable inherit that authentication check. If not enabled, it uses the fact that `SECURE_DATA_PROXY_AUTH_INCLUDE` is undefined, so the nginx config does not contain the auth request, making the data access essentially public.

`secure-data-proxy` applies in addition to file access, similar (or rather inversely) to some optional components that include extra Magpie public access rules to certain endpoints to open them.